explorer/src/pages/Mixnodes/index.tsx

```diff
@@ -39,14 +39,24 @@ export const PageMixnodes: React.FC = () => {
     }
   }, [searchTerm, mixnodes?.data])
 
+  const linkStyles = {
+    color: 'inherit',
+    textDecoration: 'none',
+    marginLeft: 16
+  }
+
   const columns = [
     {
       field: 'owner',
       headerName: 'Owner',
       width: 380,
       renderCell: (params: GridRenderCellParams) => {
         return (
-          <a href={`https://testnet-milhon-blocks.nymtech.net/account/${params.value}`} target='_blank' style={{ textDecoration: 'none', color: 'white', marginLeft: 16 }}>
+          <a
+            href={`https://testnet-milhon-blocks.nymtech.net/account/${params.value}`}
```

Comment:
Can this url go in the same place we are storing the other urls and be imported here?